person.rb

```diff
@@ -13,6 +15,11 @@ def can_use_services?
     of_age? || parent_permission == true
```

Comment:
- [ ] Let's add an instance of `Corrector` set in the `Person` class on initialization following this [lesson](
`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/composition_by_examples.md#composable-classes`). 
<img width="825" alt="Capture" src="https://user-images.githubusercontent.com/58504655/135922480-69a752e9-3225-4f4e-b0d4-b00d84610241.PNG">

Reply:
:+1:  Done!